backpack-compose/tokens.gradle.kts

```diff
@@ -167,12 +167,14 @@ tasks {
 
   val generateIcons by creating {
     this.group = group
-    project.androidFileOf("backpack-common", "src/main/res/drawable-nodpi")
-      .readAs(BpkFormat.Folder)
-      .parseAs(BpkIcon.Parser)
-      .transformTo(BpkIcon.Format.Compose(rClass))
-      .saveTo(BpkOutput.KotlinExtensionFile(src, tokensPackage, "BpkIcon"))
-      .execute()
+    doLast {
```

Comment:
this was missing - i noticed it when my task was failing on CI due to node not being installed on most jobs and noticed it was missing here too